reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreatePotentialIssuesReportRuleProvider.java

```diff
@@ -29,7 +28,7 @@
 /**
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  */
-@RuleMetadata(phase = ReportGenerationPhase.class)
+@RuleMetadata(disabled = true, phase = ReportGenerationPhase.class)
```

Comment:
Yeah, it is an NPE: http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-core-PR-dependents-check/989/consoleFull

java.lang.NullPointerException
	at org.jboss.windup.config.RuleSubset.perform(RuleSubset.java:214)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:55)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.DefaultOperationBuilder$DefaultOperationBuilderInternal.perform(DefaultOperationBuilder.java:54)
	at org.ocpsoft.rewrite.config.RuleBuilder.perform(RuleBuilder.java:168)

This happens with rare cases of nested rules (like the on in that test).